docs/user-guides/workspace-scheduling.md

```diff
@@ -37,18 +37,37 @@ days of the week your workspace is allowed to autostart.
 Use autostop to stop a workspace after a number of hours. Autostop won't stop a
 workspace if you're still using it. It will wait for the user to become inactive
 before checking connections again (1 hour by default). Template admins can
-modify the inactivity timeout duration with the
-[inactivity bump](#inactivity-timeout) template setting. Coder checks for active
-connections in the IDE, SSH, Port Forwarding, and coder_app.
+modify this duration with the "activity bump" template setting.
 
 ![Autostop UI](../images/workspaces/autostop.png)
 
-## Inactivity timeout
+## Activity detection
 
-Workspaces will automatically shut down after a period of inactivity. This can
-be configured at the template level, but is visible in the autostop description
+Workspaces automatically shut down after a period of inactivity. The "activity bump"
+duration can be configured at the template level and is visible in the autostop description
 for your workspace.
 
+### What counts as workspace activity?
+
+A workspace is considered "active" when Coder detects one or more active sessions with your workspace. Coder specifically tracks these session types:
```

Comment:
I want to add something like "saving changes to code counts as activity," which I think is true, but it seems like that might be oversimplified. Is there a good single-(noncomplex)-sentence summary we can add here?

Reply:
That would be really nice but as far as I understand our activity tracking is based on network usage and doesn't count file activity. 
I am not sure who is the right person to confirm this. 
Maybe @deansheather?

Reply:
@f0ssel I've realized I don't know what state we left things in re: the workspace activity refactor. Can you look at this section and comment whether it's accurate regarding the code as it exists today?